charts/greptimedb-cluster/values.yaml

```diff
@@ -55,8 +127,49 @@ datanode:
   # -- Extra datanode config in toml format.
   config: ""
 
-  # -- The pod template for datanode.
-  podTemplate: {}
+  # -- The pod template for datanode
+  podTemplate:
+    # -- The spec of main container
+    main:
+      # -- The datanode image.
+      image: ""
+
+      resources:
+        # -- The requested resources for the container
+        requests: {}
+        #  cpu: "1"
+        #  memory: "2Gi"
+        # -- The resources limits for the container
+        limits: {}
+        #  cpu: "1"
+        #  memory: "2Gi"
+
+      # -- The environment variables for the container
+      env: []
+
+      # -- The command to be executed in the container
+      command: []
+
+      # -- The arguments to be passed to the command
+      args: []
+
+    # -- The annotations to be created to the pod.
+    annotations: {}
+    # -- The labels to be created to the pod.
+    labels: {}
+    # -- The pod node selector
+    nodeSelector: {}
+    # -- The pod tolerations
+    tolerations: []
+    # -- The pod affinity
+    affinity: {}
+    serviceaccount:
+      # -- Create a service account
+      create: false
+      # -- The serviceaccount name
+      name: datanode-sa
```

Comment:
suggestion: The default serviceAccountName is {{ include "greptimedb-standalone.serviceAccountName" . }}, configured as follows: https://github.com/GreptimeTeam/helm-charts/blob/main/charts/greptimedb-standalone/templates/serviceaccount.yaml#L5

https://github.com/GreptimeTeam/helm-charts/blob/main/charts/greptimedb-standalone/values.yaml#L27

https://github.com/GreptimeTeam/helm-charts/blob/main/charts/greptimedb-standalone/templates/_helpers.tpl#L56

Reply:
I change to use `{{ .Release.Name }}-datanode` as the service account name and it will be more easy to read.